### ISSUE TITLE
Need to test affinity

### ISSUE THREAD

**Issue by tbbooher** [tbbooher/ekklesia#2]

**Comment by tbbooher** [comment]
and, um, understand the math

**Comment by Xqua** [comment]
The math is not really hard ! Lemme know if in need

**Comment by tbbooher** [comment]
yeah -- i understand the probability, but I don't understand how user's have a non-scalar F and S score. it seems they simply get a single score from the first test. the only repeated trials come from the bills, which receive multiple user votes, each with their own [F,S]

**Comment by Xqua** [comment]
There is a lot of way to approach this issue, one way is: you keep all the yay vote of the user, that gives you a distribution of F,S scores (the bills FS scores) which gives you the average and variance for the user (ideally you keep the distribution as non Gaussian but for simplicity we do) 

Another way which I think is better is the OK cupid approach were you have a like and dislike score based on people who voted yay and nay on a bill. So each time you vote yay or nay you get an update on one of the two distribution ! And then you can say to the user : you are X% enemy and Y% match with that bill/user/politician

**Comment by Xqua** [comment]
As for how we implemented it during the hackathon there is almost no reupdate etc. Basically what I did was to create the function that compares two point in space
